jdg/src/test/java/org/jboss/test/arquillian/ce/jdg/JdgTest.java

```diff
@@ -65,10 +60,9 @@
                 @TemplateParameter(name = "HOSTNAME_HTTPS", value="jdg-http-route.openshift"),
                 @TemplateParameter(name = "HTTPS_NAME", value="jboss"),
                 @TemplateParameter(name = "HTTPS_PASSWORD", value="mykeystorepass"),
-                @TemplateParameter(name = "IMAGE_STREAM_NAMESPACE", value="${kubernetes.namespace}")})
+                @TemplateParameter(name = "IMAGE_STREAM_NAMESPACE", value="openshift")})
```

Comment:
This removes the need of using a custom image stream resource.  I recommend installing the image stream in another project and referencing it through the namespace parameter.  Parameter values can be overridden using system variables, so no issue here.  To point to an image stream in another project: -DIMAGE_STREAM_NAMESPACE=myproject